lazy_theta_star_p_planner/src/lazy_theta_star_p.cpp

```diff
@@ -0,0 +1,262 @@
+//  Copyright 2020 Anshumaan Singh
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http:// www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+#include "lazy_theta_star_p_planner/lazy_theta_star_p.hpp"
+#include <vector>
+
+namespace lazyThetaStarP
+{
+
+LazyThetaStarP::LazyThetaStarP()
+{
+  costmap_tolerance_ = 1.75;
```

Comment:
These should be documented / parameterized for people to be able to tune if they're tunable values